src/Symfony/Bundle/FrameworkBundle/composer.json

```diff
@@ -40,7 +40,7 @@
         "symfony/dom-crawler": "~2.0,>=2.0.5",
         "symfony/intl": "~2.3",
         "symfony/security": "~2.6",
-        "symfony/form": "~2.7,>=2.7.2",
+        "symfony/form": "~2.7.23|~2.8.16",
```

Comment:
`>=3.0` does not match this new pattern whereas it matched the previous one. Is it made on purpose?